include/exadg/operators/operator_base.cpp

```diff
@@ -1354,56 +1412,63 @@ OperatorBase<dim, Number, n_components>::face_loop_diagonal(
 {
   (void)src;
 
-  IntegratorFace integrator_m =
-    IntegratorFace(matrix_free, true, this->data.dof_index, this->data.quad_index);
-  IntegratorFace integrator_p =
-    IntegratorFace(matrix_free, false, this->data.dof_index, this->data.quad_index);
-
-  // create temporal array for local diagonal
-  unsigned int const                                     dofs_per_cell = integrator_m.dofs_per_cell;
-  dealii::AlignedVector<dealii::VectorizedArray<Number>> local_diag(dofs_per_cell);
-
-  for(auto face = range.first; face < range.second; ++face)
+  if(is_dg)
```

Comment:
I think this is a mixture of different styles. In other places, we use functions `face_loop_empty()` so that we see on the top level (`matrix_free->loop()/cell_loop()`) which type of loops we have. I think we need a consistent way, otherwise we can here and there introduce if-statements "for safety". So I think we should not write `if(is_dg)` here.